orchagent/dash/dashorch.cpp

```diff
@@ -31,10 +32,38 @@ extern sai_dash_eni_api_t* sai_dash_eni_api;
 extern sai_object_id_t gSwitchId;
 extern size_t gMaxBulkSize;
 extern CrmOrch *gCrmOrch;
+extern bool gTraditionalFlexCounter;
 
-DashOrch::DashOrch(DBConnector *db, vector<string> &tableName, ZmqServer *zmqServer) : ZmqOrch(db, tableName, zmqServer)
+#define FLEX_COUNTER_UPD_INTERVAL 1
```

Comment:
Why there are two defines for the update interval? What is the difference comparing to `ENI_STAT_FLEX_COUNTER_POLLING_INTERVAL_MS`?

Reply:
It's redundant, will remove

Reply:
Actually i'm wrong sorry. FLEX_COUNTER_UPD_INTERVAL is the period for SelectableTimer. ENI_STAT_FLEX_COUNTER_POLLING_INTERVAL_MS is the default polling interval for FlexCounter i.e. to  collect and populate the COUNTERS_DB